### ISSUE TITLE
LaTexTools Logistic Question

### ISSUE THREAD

**Issue by suyunbae** [SublimeText/LaTeXTools#853]
I hope this is the correct place to ask this question, but I am needing some help setting up the LaTex Tools plugin for Sublime Text 3. I am currently using Build 3126. My professor encouraged us to start using LaTex for our homework and paper and I have been trying to add it into my text editor but I keep getting errors. I have two questions to ask if anybody can help

1.  As soon as I try to open a .tex file sublime gives me an error stating: 
    ```
    Error loading syntax file "Packages/LaTeX/LaTeX.sublime-syntax": Unable to read Packages/LaTeX/LaTeX.sublime-syntax
    ```

2. When I try to compile this said .tex file I get an error in the compiler stating:
    ```
    [Compiling /Users/Sandra/Desktop/first.tex]

    TraditionalBuilder: Engine: pdflatex. Invoking latexmk...

    COULD NOT COMPILE!

    Attempted command:latexmk -cd -f -pdf -interaction=nonstopmode -synctex=1 first.tex
    Build engine: Traditional Builder
    ```

Could anyone please provide an insight as to how I can fix this?

**Comment by r-stein** [comment]
1. That the LaTeX Syntax is not working for you is pretty weird. You could try to go to https://github.com/sublimehq/Packages, download the repository, and put the `LaTeX` folder in your packages folder `Preferences >> Browse Packages...`
2. After the build failed open the Sublime Text Console via `View >> Show Console` and copy the error message. You may also try to run the command from a shell and look if it also fails.

**Comment by suyunbae** [comment]
Hi @r-stein ! 

Thank you so much! I fixed the issues with your support! The reason why I couldn't build the .tex file was because the compiler could not find latexmk. I remember the reason why I couldn't download BasicTex was because I kept getting an error saying that I needed to update my tlmgr in Terminal. I finally found a way to do that and everything is working smoothly now. Thank you so much for your help!
